src/borg/testsuite/archiver/debug_cmds.py

```diff
@@ -86,7 +86,7 @@ def test_debug_id_hash_format_put_get_parse_obj(self):
             "input/plain.bin",
             "input/meta.json",
             "output/data.bin",
-            "--compression=zstd,2",
+            "--compression=zstd,3",
```

Comment:
No, please keep level at 2.

3 is the default zstd level and we want something special (non-standard) to detect.